### PR TITLE
Warning storing private key for the template

### PR DESCRIPTION
**Pull request by julienbrs**
closes #134
I used the `:::warning:::` docusaurus flag to wrap the message, maybe it appears too aggressive ?

### CODE REVIEW

File: docs/build/quickstart.mdx [hunk 1]
```diff
@@ -275,10 +275,16 @@ function Content() {
...
 
-:::caution
+:::info
 Make sure to replace `SECRET_KEY` and `SC_ADDRESS` with your own secret key and smart contract address.
 :::
 
+:::warning
+Because this template is storing your secret key in plain text, it's not secure and is only used for demonstration purposes.
+Your secret key is a highly sensitive piece of information and should never be shared with anyone. If you're 
+using a public repository, make sure to add your secret key to a .env file and add it to your .gitignore file.
```
Comment (qdrn) [inline docs/build/quickstart.mdx:285]: It's in the front part. So I don't think the `.env` is relevant. Better mention wallets. Or nothing as it's common knowledge ?

**Review by qdrn — CHANGES_REQUESTED** [review]
I agree that warning might be too much given that it's quite obvious it's a basic example ? Maybe caution is enough ?

File: docs/build/quickstart.mdx (hunk 1 above)
Comment (AurelienFT) [reply docs/build/quickstart.mdx:285]: Yes the key is in the .ts directly not in the .env and mentioning wallets is great.

Comment (AurelienFT) [reply docs/build/quickstart.mdx:285]: Yes but the key here in the front side example is directly in .ts so there is no .env

Comment (julienbrs) [reply docs/build/quickstart.mdx]: Indeed my bad! I've mentionned the wallet provider

**Review by qdrn — APPROVED** [review]

**Pull request merged**